web3.md

```diff
@@ -186,5 +193,37 @@ module WEB3
            <code> CODE </code>
            ...
          </account>
+
+    syntax KItem ::= "#evm_snapshot"
+ // --------------------------------
+    rule <k> #evm_snapshot => #pushNetworkState ~> #sendResponse( "result" : #unparseQuantity( size ( SNAPSHOTS ) ) ) ... </k>
+         <snapshots> SNAPSHOTS </snapshots>
+
+    syntax KItem ::= "#pushNetworkState"
+ // ------------------------------------
+    rule <k> #pushNetworkState => . ... </k>
+         <snapshots>... .List => ListItem (NETWORKSTATE) </snapshots>
+         <network> NETWORKSTATE </network>
+
+    syntax KItem ::= "#evm_revert"
+ // ------------------------------
+    rule <k> #evm_revert => #sendResponse( "result" : "true" ) ... </k>
+         <params> [.JSONList] </params>
+         <snapshots>... ListItem(NETWORKSTATE) => .List </snapshots>
+         <network> _ => NETWORKSTATE </network>
+
+    rule <k> #evm_revert ... </k>
+         <params> [ (DATA => #parseHexWord(DATA))] </params>
+
+    rule <k> #evm_revert => #sendResponse( "result" : "true" ) ... </k>
+         <params> [DATA, .JSONList ] </params>
+         <snapshots> SNAPSHOTS => range(SNAPSHOTS, 0, DATA) </snapshots>
+         // <network> NETWORKSTATE => (SNAPSHOTS [DATA]) </network>
```

Comment:
Return type of `_[_]` is `KItem`. `{SNAPSHOTS [ DATA ]}:>NetworkCellFragment` (or `NetworkCell`).